hw-12/main.js

```diff
@@ -0,0 +1,37 @@
+// Exercise 1:
+const checkIsEqual = (firstArr, secondArray) => {
+  return JSON.stringify(firstArr) === JSON.stringify(secondArray)
```

Comment:
Vd:
```javascript
  const arr1 = [1, 2]
  const arr2 = [2, 1]
  
  // sau khi stringify
  const str1 = "[1, 2]"
  const str2 = "[2, 1]"
```
So sánh vẫn phải trả về `arr1` is equal `arr2`

Nếu stringify như bên trên sẽ bị sai

Reply:
ban đầu e dùng vòng for, sau đó thử dùng every() đều được nhưng e thấy cách này ngắn nhất nên đưa vào bài, để e sửa lại.